src/actions/itemActionType.js

```diff
@@ -6,3 +6,4 @@ export const ITEM_API_ERROR = 'ITEM_API_ERROR'
 export const GET_SHARED_ITEMS_PENDING = 'GET_SHARED_ITEMS_PENDING'
 export const VIEWING_SHARED_ITEMS = 'VIEWING_SHARED_ITEMS'
 export const GET_STARRED_ITEMS_PENDING = 'GET_STARRED_ITEMS_PENDING'
+export const CREATE_FILEMANAGER_PENDING = 'CREATE_FILEMANAGER_PENDING'
```

Comment:
make seperate actions, reducers,actiontypes for filemanager as it is a completely different model